runtime-sdk/src/modules/accounts/mod.rs

```diff
@@ -69,18 +79,29 @@ pub enum Event {
     },
 }
 
+/// Gas costs.
+#[derive(Clone, Debug, Default, Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct GasCosts {
+    #[serde(rename = "tx_transfer")]
+    pub tx_transfer: u64,
+}
+
 /// Parameters for the accounts module.
 #[derive(Clone, Debug, Serialize, Deserialize)]
 #[serde(deny_unknown_fields)]
 pub struct Parameters {
     #[serde(rename = "transfers_disabled")]
     pub transfers_disabled: bool,
+    #[serde(rename = "gas_cost")]
```

Comment:
```suggestion
    #[serde(rename = "gas_costs")]
```

Reply:
ah yes, here and in the key value runtime